pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentAggregationMultiValueQueriesTest.java

```diff
@@ -519,6 +519,210 @@ public void testPercentileEst99MV() {
         .testInterSegmentAggregationResult(brokerResponse, 400000L, 0L, 800000L, 400000L, new String[]{"2147483647"});
   }
 
+  @Test
+  public void testPercentileRawEst50MV() {
```

Comment:
We should be able to refactor these tests so that we don't have to write the same code for each of the percentile value test right?

Reply:
Most of the tests in this file use this skeleton. There was already a refactor to use testInterSegmentAggregationResult() which simplified this a bit. We should be able to refactor it further by passing the query, expected values etc.

Reply:
Refactored a bit to avoid duplicate code.

Reply:
Done.